scripts/templates/grafana-dashboards/dm-apps.json

```diff
@@ -0,0 +1,806 @@
+        "multi": false,
+        "name": "appRelease",
+        "options": [
+          {
+            "selected": true,
+            "text": "api-release",
+            "value": "api-release"
+          },
+          {
+            "selected": false,
+            "text": "search-api-release",
+            "value": "search-api-release"
+          },
+          {
+            "selected": false,
+            "text": "buyer-frontend-release",
+            "value": "buyer-frontend-release"
+          },
+          {
+            "selected": false,
+            "text": "supplier-frontend-release",
+            "value": "supplier-frontend-release"
+          },
+          {
+            "selected": false,
+            "text": "admin-frontend-release",
+            "value": "admin-frontend-release"
+          },
+          {
+            "selected": false,
+            "text": "briefs-frontend-release",
+            "value": "briefs-frontend-release"
+          }
+        ],
+        "query": "api-release,search-api-release,buyer-frontend-release,supplier-frontend-release,admin-frontend-release,briefs-frontend-release",
+        "type": "custom"
+      }
+    ]
+  },
+  "annotations": {
+    "list": []
+  },
+  "refresh": "1m",
+  "schemaVersion": 13,
+  "version": 33,
+  "links": [],
+  "gnetId": null,
+  "rows": [
+    {
+      "title": "Dashboard Row",
+      "panels": [
+        {
+          "aliasColors": {
+            "0-0.025s": "#3F6833",
+            "0.025-0.05s": "#629E51",
+            "0.05-0.1s": "#7EB26D",
+            "0.1-0.25s": "#F2C96D",
+            "0.25-0.5s": "#EAB839",
+            "0.5-1s": "#EF843C",
+            "1-2.5s": "#C15C17",
+            "2.5-5s": "#E24D42",
+            "5-10s": "#890F02",
+            ">10s": "#58140C"
+          },
+          "bars": true,
+          "datasource": null,
+          "editable": true,
+          "error": false,
+          "fill": 1,
+          "id": 1,
+          "legend": {
+            "alignAsTable": true,
+            "avg": false,
+            "current": false,
+            "hideEmpty": false,
+            "hideZero": false,
+            "max": false,
+            "min": false,
+            "rightSide": true,
+            "show": true,
+            "total": true,
+            "values": true
+          },
+          "lines": false,
+          "linewidth": 1,
+          "links": [],
+          "nullPointMode": "connected",
+          "percentage": false,
+          "pointradius": 5,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [
+            {
+              "alias": "0-0.1s",
+              "yaxis": 1
+            }
+          ],
+          "span": 12,
+          "stack": true,
+          "steppedLine": false,
+          "targets": [
+            {
+              "hide": false,
+              "refId": "A",
+              "target": "alias(cloudwatch.request_time_buckets.$environment.$app.request_time_bucket_0.samplecount, '0-0.025s')"
```

Comment:
So using `sum` or `avg` on these is for an individual metric at a timestamp rather than any aggregation of a metric over a period of time.

For example if we have
```
requests_bucket_1 50 1500031162
requests_bucket_1 100 1500031162
```
then using `sum`  would give us
```requests_bucket_1 150 1500031162```
or using `avg` would give us
```requests_bucket_1 75 1500031162```

We did had a problem that we were sending different values for the same metric for the same timestamp (first an incomplete value and then the full value) so would make a difference if we chose sum or avg. However, we have fixed this so there is no duplicated metric data being sent and therefore it should not matter if we choose sum or avg. 

Therefore it feels like we are fine to keep avg. If we really wanted to cover the case were we would have double metrics being sent per timestamp then we could use the `max` filter.